.cirrus.yml

```diff
@@ -101,6 +101,26 @@ fedora_packaging_task:
 
     timeout_in: '20m'
 
+# Verify calls to bin/config were saved
+config_task:
```

Comment:
Need to set up dependencies here and in the other tasks (to depend on this one passing).

Reply:
Let's do:

```yaml
config_task:

    depends_on:
        - "gating"
```

then add `config` (no "task" suffix) into the dependency lists for:

* meta_task
* testing_task
* success_task

Reply:
we don't a gating task as of now, will this change add one?

Reply:
nor meta, testing, or success tasks. I could add the dependency to integration and and fedora_packaging tasks?